plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -98,9 +99,17 @@ public BigQueryConfig setViewsEnabled(boolean viewsEnabled)
         return this;
     }
 
-    public Duration getViewExpiration()
+    @NotNull
+    public Duration getViewExpireDuration()
```

Comment:
Some reasonable MinDuration?

Reply:
I considered the annotation before sending this PR, but I didn't come up with the reasonable value. 5 minutes?

Reply:
This should be > view cache ttl?

Reply:
Let me add a validation method.

Reply:
@hashhar Updated.